tests/test_easing.h

```diff
@@ -0,0 +1,163 @@
+		{300.000000,308.402771,333.611115,375.625000,434.444458,510.069458,527.500000,499.236115,487.777771,493.125000,515.277771,547.986084,535.000000,538.819458,546.944458,550.000000,553.055542,561.180542,565.000000,552.013855,584.722229,606.875000,612.222229,600.763855,572.499939,589.930664,665.555664,724.375061,766.388916,791.597229,800.000000,},
+	},
+	{
+		{300.000000,299.104706,296.618958,292.842896,288.076721,282.620605,276.774719,270.839233,265.114349,259.900238,255.497040,252.204956,250.324158,250.154831,251.997131,256.151245,262.917358,272.595642,285.486237,301.889374,322.105194,346.433899,375.175598,408.630615,447.098938,490.880829,540.276611,595.586121,657.109741,725.147705,800.000000,},
+		{300.000000,374.852478,442.890259,504.414062,559.723511,609.119263,652.901123,691.369507,724.824402,753.566162,777.894836,798.110657,814.513794,827.404358,837.082642,843.848755,848.002869,849.845154,849.675842,847.795044,844.502991,840.099731,834.885620,829.160767,823.225220,817.379395,811.923218,807.157104,803.381104,800.895264,800.000000,},
+		{300.000000,297.383057,290.597382,281.240723,270.910797,261.205383,253.722168,250.058929,251.813370,260.583252,277.966339,305.560303,344.962921,397.771973,465.585083,550.000000,634.415039,702.228210,755.037109,794.439819,822.033691,839.416809,848.186584,849.941040,846.277832,838.794617,829.089172,818.759277,809.402588,802.616943,800.000000,},
+		{300.000000,371.445129,429.861755,476.450562,512.412231,538.947388,557.256897,568.541321,574.001465,574.837891,572.251465,567.442810,561.612610,555.961609,551.690552,550.000000,548.309448,544.038330,538.387329,532.557190,527.748535,525.162109,525.998535,531.458679,542.743164,561.052612,587.587891,623.549500,670.138306,728.554871,800.000000,},
+	},
+};
+/* clang-format on */
+
+TEST_CASE("[Easing] Test easing functions results") {
+	MockTween tween;
+	// Current time, initial_val, delta_val, duration
+	float results[Tween::TRANS_COUNT][Tween::EASE_COUNT][STEPS + 1] = {};
+	for (int tr = 0; tr < Tween::TRANS_COUNT; tr++) {
+		for (int ease = 0; ease < Tween::EASE_COUNT; ease++) {
+			for (int i = 0; i <= STEPS; i++) {
+				Tween::TransitionType tt = (Tween::TransitionType)tr;
+				Tween::EaseType et = (Tween::EaseType)ease;
+				const float t = TIME_START + (TIME_END - TIME_START) / STEPS * i;
+				real_t v = tween.run_equation(tt, et, t, 300, 500, TIME_END);
+				results[tr][ease][i] = v;
+			}
+		}
+	}
+#if 0
+	// Print the results array.
+	printf("{\n");
+	for (int x = 0; x < Tween::TRANS_COUNT; x++) {
+		printf("\t{\n");
+		for (int y = 0; y < Tween::EASE_COUNT; y++) {
+			printf("\t\t{");
+			for (int z = 0; z <= STEPS; z++) {
+				printf("%f,", results[x][y][z]);
+			}
+			printf("},\n");
+		}
+		printf("\t},\n");
+	}
+	printf("\n};\n");
+#endif
```

Comment:
Is it relevant to keep the `#if 0`'ed code?

Reply:
well, if we ever need to regenerate the array.
But it could also easily be rewritten, so I can remove it if you prefer.

Reply:
Well I guess it doesn't hurt to keep, maybe just improve the comment to explain its purpose (i.e to re-create the results array if there's ever a change to the behavior of Tween and we want to update the test).